### PR TITLE
Fix shopify debug typing

### PR DESCRIPTION
**Pull request by nanaya**
More like working around it because a proper fix of maybe importing `@types/shopify-buy` seems quite involved.

### CODE REVIEW

**Comment by notbakaneko** [comment]
`@types/shopify-buy` doesn't even have the correct types 🙃

**Comment by nanaya** [comment]
maybe should look into fixing it upstream :thinking: the `checkout.create` is very wrong at the moment.

File: resources/assets/lib/shopify-debug.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export async function fetchAllVariants(): Promise<any[]> {
   return variants;
 }
 
-export async function fetchAllVariantIds(): Promise<{ }> {
+export async function fetchAllVariantIds(): Promise<unknown[]> {
```
Comment (notbakaneko) [inline resources/assets/lib/shopify-debug.ts:39]: The explicit return type isn't needed (it's `{ gid: string, name: string }`,  anyway)

**Review by notbakaneko — CHANGES_REQUESTED** [review]

File: resources/assets/lib/shopify-debug.ts (hunk 1 above)
Comment (nanaya) [reply resources/assets/lib/shopify-debug.ts:39]: it is required...

```
TS2314: Generic type 'Promise<T>' requires 1 type argument(s).
```

Comment (nanaya) [reply resources/assets/lib/shopify-debug.ts:39]: oh I forgot about just removing the typing entirely :raised_eyebrow:

**Review by notbakaneko — APPROVED** [review]

**Pull request merged**